packages/liveblocks-client/.eslintrc.js

```diff
@@ -6,6 +9,7 @@ module.exports = {
     "@typescript-eslint",
     "eslint-plugin-import",
     "eslint-plugin-simple-import-sort",
+    "eslint-plugin-rulesdir",
```

Comment:
The "rulesdir" ESLint plugin allows us to add custom ESLint rules with syntax rules we want to apply to this code base.